src/index.html

```diff
@@ -15,18 +16,50 @@
       rel="stylesheet"
       href="style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
```

Comment:
The URL for the href attribute of the link tag seems to be missing. Ensure you provide the correct path to the stylesheet.